.github/workflows/release.yml

```diff
@@ -76,12 +76,54 @@ jobs:
           cp ./semgrep-ubuntu/artifacts.tar.gz.sha256 semgrep-${{ steps.get_version.outputs.VERSION }}-ubuntu-16.04.tgz.sha256
           gh release --repo returntocorp/semgrep upload ${{ steps.get_version.outputs.VERSION }} semgrep-${{ steps.get_version.outputs.VERSION }}-ubuntu-16.04.tgz.sha256
           rm semgrep-${{ steps.get_version.outputs.VERSION }}-ubuntu-16.04.tgz.sha256
+      - uses: actions/download-artifact@v3
+        with:
+          name: rule-schema-json
+          path: rule_schema.json
+      - uses: actions/download-artifact@v3
+        with:
+          name: rule-schema-yaml
+          path: rule_schema.yaml
+      - name: Upload Schema Files
+        id: upload-semgrep-schema-files
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh release --repo returntocorp/semgrep-interfaces upload ${{ steps.get_version.outputs.VERSION }} rule_schema.json
```

Comment:
I don't know if this will work without having created a release on that repository prior

Reply:
Typically a release is created off a tag, so we may need to also create a tag in `returntocorp/semgrep-interfaces`.

Reply:
Yea looks like we'll have to